docs/cloud/cli/connector.mdx

```diff
@@ -93,7 +93,7 @@ Example usage:
 ```yaml
 apiVersion: 0.1.0
 meta:
- version: 0.2.5
```

Comment:
at this: 

:::info
All versions are marked with `x.y.z`. To find the latest version, run:
*  `fluvio hub connector list`
*  `fluvio hub smartmodule list`
:::

Reply:
added